ankihub/gui/enable_fsrs_dialog.py

```diff
@@ -0,0 +1,84 @@
+from datetime import date
+from typing import Optional
+
+import aqt
+
+from .. import LOGGER
+from ..settings import ANKI_INT_VERSION, config
+from .deck_options import MIN_ANKI_VERSION_FOR_FSRS_FEATURES
+from .utils import show_dialog
+
+ENABLE_FSRS_REMINDER_INTERVAL = 30
```

Comment:
:pick: I think we should rename this to make it clear what the unit is, and also to make it consistent with `FSRS_OPTIMIZATION_REMINDER_INTERVAL_DAYS`:
```suggestion
ENABLE_FSRS_REMINDER_INTERVAL_DAYS = 30
```